icons/book-open-check.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M2 3h6c2.2 0 4 1.8 4 4v14c0-1.7-1.3-3-3-3H2V3z"></path>
+  <path d="m16 12 2 2 4-4"></path>
+  <path d="M18 18h-3c-1.7 0-3 1.3-3 3v-9m10-6h0V3h-6c-2.2 0-4 1.8-4 4v5m6 6h4v-2.3"></path>
```

Comment:
```suggestion
  <path d="M8 3H2v15h7c1.7 0 3 1.3 3 3V7c0-2.2-1.8-4-4-4Z" />
  <path d="m16 12 2 2 4-4" />
  <path d="M22 6V3h-6c-2.2 0-4 1.8-4 4v14c0-1.7 1.3-3 3-3h7v-2.3" />
```
There were a few unnecessary points in one of the paths, I've put it through svgo.

Reply:
Thank you, copied the original one `book-open`, so it's possible that there are also unnecessary points in the paths.